src/main/java/seedu/club/model/ClubBook.java

```diff
@@ -520,4 +535,21 @@ public void updateTask(Task taskToEdit, Task editedTask) throws DuplicateTaskExc
         requireNonNull(editedTask);
         tasks.setTask(taskToEdit, editedTask);
     }
+
+    /**
+     * Removes all tasks that have been assigned to {@code member}.
+     */
+    public int removeTasksOfMember(Member member) {
+
+        int numberOfTasksRemoved = 0;
+        Iterator<Task> it = tasks.iterator();
+        while (it.hasNext()) {
+            Task task = it.next();
+            if (task.getAssignee().getAssignee().equalsIgnoreCase(member.getMatricNumber().toString())) {
```

Comment:
Idk about this but, maybe you should consider making 
`boolean isTaskAssignedToMember`

Reply:
alright. will do 👍